src/patternfly/components/ToggleGroup/examples/ToggleGroup.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Toggle group
+section: components
+cssPrefix: pf-c-toggle-group
+beta: true
+---
+
+import './ToggleGroup.css'
+
+## Examples
+```hbs title=Default
+{{#> toggle-group}}
+  {{#> toggle-group-button}}
+    {{#> toggle-group-text}}
+      Option 1
+    {{/toggle-group-text}}
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button}}
+    {{#> toggle-group-text}}
+      Option 2
+    {{/toggle-group-text}}
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button}}
+    {{#> toggle-group-text}}
+      Option 3
+    {{/toggle-group-text}}
+  {{/toggle-group-button}}
+{{/toggle-group}}
+
+<br/>
+
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-selected"}}
+    {{#> toggle-group-text}}
+      Option 1
+    {{/toggle-group-text}}
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-selected"}}
```

Comment:
```suggestion
  {{#> toggle-group-button}}
```
Looks like you have two set to selected

Reply:
this is to demonstrated two things can be clicked.